### PR TITLE
Feature / reservation changes

### PR DESCRIPTION
**Pull request by SedlarDavid**

### CODE REVIEW

File: app/Http/Controllers/ReservationController.php [hunk 1]
```diff
@@ -111,32 +112,70 @@ public function index()
...
-        // ]);
+        $request->validate([
+            'name' => 'required',
+            'email' => 'required',
+            'tel' => 'required',
+            'consent' => 'required'
+        ]);
+        $seatsData =  json_decode($request->input('seats'), true);
+
+        if ($request->input('stand') == 0 &&  count($seatsData ?? []) == 0) {
```
Comment (tomesfilip) [inline app/Http/Controllers/ReservationController.php:123]: is the double space after double && okay? the same after` $seatsData = `

File: app/Http/Controllers/ReservationController.php [hunk 2]
```diff
@@ -111,32 +112,70 @@ public function index()
...
+    }
+    private function getSeatsPrice($seats)
+    {
+
+        $priceSit = 500;
+        $priceSitRaut = 750;
+
+        $totalPrice = 0;
+        foreach ($seats as $seat) {
+            if ($seat['typ'] == "raut") {
```
Comment (tomesfilip) [inline app/Http/Controllers/ReservationController.php:170]: is there a reason to use double quotes for `"raut"` ?

**Review by tomesfilip — COMMENTED** [review]
gj, just neatpicks 👍

File: app/Http/Controllers/ReservationController.php (hunk 2 above)
Comment (SedlarDavid) [reply app/Http/Controllers/ReservationController.php]: no reason at all, replaced with singe quotes

File: app/Http/Controllers/ReservationController.php (hunk 1 above)
Comment (SedlarDavid) [reply app/Http/Controllers/ReservationController.php]: just formatting issue, no impact on functionality

**Review by tomesfilip — APPROVED** [review]
gj

**Pull request merged**